Tracker/Modules/OnboardingModule/OnboardingFirstEnterChecker.swift

```diff
@@ -20,12 +21,12 @@ final class OnboardingFirstEnterChecker {
 }
 
 extension OnboardingFirstEnterChecker: FirstEnterCheckableProtocol {
-    func didEnter() -> Bool {
-        if onboardingFirstEnterStorage.didEnterBefore == true {
-            return false
-        } else {
-            onboardingFirstEnterStorage.didEnterBefore = true
-            return true
-        }
+    func shouldShowOnboarding() -> Bool {
+        onboardingFirstEnterStorage.hasEnteredBefore ? false : true
```

Comment:
буль превращающийся в тернарник с булями – это перебор :) 

```suggestion
        !onboardingFirstEnterStorage.hasEnteredBefore
```

Reply:
Как всегда, лучшее ревью - у тебя. Спасибо большое!

Reply:
Спасибо!